dgl_ptm/dgl_ptm/util/matrix_utils.py

```diff
@@ -0,0 +1,72 @@
+"""This module contains functions for DGL matrix manipulations."""
+
+import torch
+import dgl
+
+
+def apply_mask_to_sparse_matrix(sparse_matrix, mask):
+    """
+    Apply a mask to a sparse matrix and return an appropriately masked sparse matrix.
+
+    Args:
+        sparse_matrix (dgl.sparse.SparseMatrix): the original sparse matrix.
+        mask (tensor): the mask to be applied.
+    
+    Return (dgl.sparse.SparseMatrix) the masked matrix.
```

Comment:
I noticed that different styles of docstrings are used everywhere in dgl_ptm like [google style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstring i.e. `Args:` whereas the [reST style](https://peps.python.org/pep-0287/) i.e. `param:`. Can we make them consistent in `matrix_utils.py`? The rest can be fixed as a part of #37

Reply:
I assume that we are not using Sphinx to generate documentation, so I vote for google style which is more simple.